### PR TITLE
Installation guide

### PR DESCRIPTION
**Pull request by marcusolsson**
This PR adds a new top-level category "Getting started" with two new pages:

- **Download and install Obsidian** contains guides for installing Obsidian on every supported platform.
- **Update Obsidian** contains guides for managing updates and checking the current version

Was a little unsure about how to recommend users to update the installer version. Most users probably don't know what a "minor" version is. Maybe we could recommend a time period instead, e.g. "every other month"? 

> If the installer version differs from the current version by several minor versions, consider reinstalling Obsidian.

### CODE REVIEW

**Comment by ericaxu** [comment]
> Was a little unsure about how to recommend users to update the installer version. Most users probably don't know what a "minor" version is. Maybe we could recommend a time period instead, e.g. "every other month"?

As Electron gets more stable I think every other month might be too much. Does it make sense to recommend doing it on a per need basis? Like if something is not working as intended, consider updating that first.

0.13 (in beta right now) introduces a way to show debug info, which takes care of this case.

**Comment by ericaxu** [comment]
Looks good otherwise and long overdue! I think it's especially useful on the web help version (because if you have help vault you've already installed it).

**Comment by marcusolsson** [comment]
> As Electron gets more stable I think every other month might be too much. Does it make sense to recommend doing it on a per need basis? Like if something is not working as intended, consider updating that first.

If we can define that need I think that's definitely to be preferred. Sounds like users shouldn't have to care about the installer version unless they run into an issue. WDYT about moving this to a Troubleshooting section instead then?

**Comment by ericaxu** [comment]
> > As Electron gets more stable I think every other month might be too much. Does it make sense to recommend doing it on a per need basis? Like if something is not working as intended, consider updating that first.
> 
> If we can define that need I think that's definitely to be preferred. Sounds like users shouldn't have to care about the installer version unless they run into an issue. WDYT about moving this to a Troubleshooting section instead then?

That sounds great. Something along the lines of "if a feature or fix is supposed to work but don't, update Obsidian first. If that's still not resolved, consider reinstalling."

**Comment by marcusolsson** [comment]
@ericaxu I've rebased and addressed your feedback. Let me know if you think it needs more work!

**Comment by ericaxu** [comment]
> @ericaxu I've rebased and addressed your feedback. Let me know if you think it needs more work!

Thanks! Could you confirm that not having the ending period after external links is intentional because of the external link icon?

File: en/Obsidian/Official website.md [hunk 1]
```diff
@@ -2,6 +2,6 @@ Obsidian’s official website is at https://obsidian.md/.
 
 It’s useful for:
 
-1. Grabbing the latest installer if your [[Installer version]] is too low;
-2. Register an account if you want to purchase [[Catalyst license]], [[Commercial license]], [[Introduction to Obsidian Sync|Obsidian Sync]], or [[Introduction to Obsidian Publish|Obsidian Publish]];
-3. Playing a [quiz game](https://obsidian.md/quiz) if you’re confident with your knowledge of the community plugins and themes.
\ No newline at end of file
+1. Grabbing the latest installer if your installer version is too low;
```
Comment (frozenfossill) [inline en/Obsidian/Official website.md:5]: I guess it should be `if your installer version is too old`?

**Comment by marcusolsson** [comment]
> Thanks! Could you confirm that not having the ending period after external links is intentional because of the external link icon?

Not intentional. I've added the missing periods.

**Pull request merged**